doc/classes/String.xml

```diff
@@ -744,6 +744,14 @@
 				Replaces all occurrences of [param what] inside the string with the given [param forwhat].
 			</description>
 		</method>
+		<method name="replace_all" qualifiers="const">
```

Comment:
In my opinion, the name `replace_multiple` would be better, since otherwise the user might think that `replace_all()` is a version of `replace()`. But `replace()` already replaces all occurrences, not just the first one (like in some languages/libraries).

We should also think about naming if we decide to add a pair replacement method in the future (`replace_*(["a", "b"], ["x", "y"])` or `replace_*({"a": "x", "b": "y"})`).

Reply:
Might also be good to clarify that it does not replace recursively, i.e. `"banana".replace_all(["b","na"],"x")` gives `"xaxx"` not `"xxx"`